src/rgw/rgw_sal_motr.cc

```diff
@@ -1034,11 +1035,37 @@ int MotrBucket::list(const DoutPrefixProvider *dpp, ListParams& params, int max,
       rgw_bucket_dir_entry ent;
       auto iter = vals[i].cbegin();
       ent.decode(iter);
+      std::string key_name = ent.key.name + "/null";
+      if(keys[i] == key_name)
+      { 
+        // read data for null_index entry - (obj1/null)
+        bufferlist bl_null;
+        ret_code = this->store->do_idx_op_by_name(bucket_index_iname,
+                                    M0_IC_GET, key_name, bl_null);
+        ldpp_dout(dpp, 2) << "**** null key found for object -"<< keys[i] << dendl;
+        // read value for null index key(obj1/null)
+        ldpp_dout(dpp, 20) <<__func__<< "****** GET null rc : " << rc << dendl;
```

Comment:
cleanup log statements added newly and remove which are not needed. like the one here or line 1062 log definitely needed change

Reply:
done